http_client.rst

```diff
@@ -215,7 +215,7 @@ autoconfigure the HTTP client based on the requested URL:
             http_client:
                 scoped_clients:
                     # only requests matching scope will use these options
-                    github:
+                    github.client:
```

Comment:
github_client:

Works the same 👍🏻

Reply:
@OskarStark I'm not sure what you mean? Do you think I should rename all `github.client` as `github_client`? What is the point?

Reply:
I see yaml keys more often with underscores. Sure github.client works fine, not sure what to promote

cc @nicolas-grekas